news/2024/2024-01-26-mwc-7k-ro24-summary.md

```diff
@@ -0,0 +1,83 @@
+
+Russia had an obvious getaway pick with [shadow\_bling - Crush Alcohol](https://osu.ppy.sh/beatmapsets/2062870#mania/4452380) — but misses from both sides kept this close until the Russians barely pulled away in the end of the chart. Denmark went for [seatrus - Raindrop](https://osu.ppy.sh/beatmapsets/2119535#mania/4452365) as a counter, with Jole coming into this chart as one of the formidable chordstream players in this round. They definitely showed up with a 5x 200 score on Raindrop tying up the scoreline yet again.
+
+Both teams were put in a very difficult position given the remaining charts in the pool, ending up in a tiebreaker scenario on [Massive New Krew - Majestic 12](https://osu.ppy.sh/beatmapsets/2119554#mania/4452392), that resulted in the Russians winning the match 5—4.
+
+### ::{ flag=GB }:: United Kingdom vs. ::{ flag=DE }:: Germany
+
+[**View the VOD here!**](https://www.twitch.tv/videos/2039449910)
+
+The United Kingdom versus Germany match was a sight to behold! With similarly placed seeds, everyone expected everyone to fight tooth and nail in order to gain an advantage over their rivals.
+
+From the very beginning, the teams didn't fear to show their preferred skill sets. While the United Kingdom banned a Long Note pick and picked a Rice one subsequently, Germany did the exact opposite. These biases were shown all throughout the match, as these teams stuck to their strong suits.
+
+The match then began with [seatrus - Raindrop](https://osu.ppy.sh/beatmapsets/2119535#mania/4452365) where Germany managed to squeeze their first point thanks to their consistency despite the early miss. Little did they know this aforementioned consistency was going to be what would uplift their lead for the rest of the match! The advantage Germany gained thanks to their LN picks and ::{ flag = DE }:: [TsukiyaWhiskers](https://osu.ppy.sh/users/12598261)' amazing performance throughout the match would unfortunately seal the deal as the United Kingdom was not able to maintain their leads.
+
+Alas, this was no stomp by any stretch of the imagination. The United Kingdoms's performance was admirable. And while they weren't able to secure a point, they managed to put Germany on check more than a few times. And [Y.W - Freshness Vacation](https://osu.ppy.sh/beatmapsets/2119550#mania/4452388) was a testament to that.
+
+Shout-outs to ::{ flag = DE }:: [sinkip](https://osu.ppy.sh/users/4373641) for playing the entire match [with a BMS controller](https://www.twitch.tv/videos/2039250337?t=04h05m49s)!
+
+---
+
+Things are already heating up in this year's rendition of the MWC7K! The North American clash from OWC's Quarterfinals between the USA and Canada is continuing on in MWC7K, so that should be fun to watch. Australia also has some returning players that will surely have an impact on their match versus Brazil, so be sure to check that match out as well!
```

Comment:
minor but thoughts on also using flags next to country names here?

Reply:
Hasn't been done when they were mentioned in the highlights. Though I understand having them in the conclusion would be more eye catching than without it.

Reply:
no <.<

Reply:
No it is.